contracts/feeManager/FeeManager.sol

```diff
@@ -131,12 +131,7 @@ contract FeeManager is IFeeManager, Ownable {
             remainingFunds = _payOwner(debtor, remainingFunds);
 
             if (remainingFunds > 0) {
-                (bool success, ) = msg.sender.call{value: remainingFunds}("");
-
-                if (!success) {
-                    // If the msg.sender can't receive the remaining funds, store them for future withdrawal
-                    _funds[msg.sender] += remainingFunds;
-                }
+                _funds[msg.sender] += remainingFunds;
```

Comment:
I also like the idea of returning remaining funds, but this was the only way we found to avoid reentrancy attacks for now, but we can discuss if there is another way to achieve both, returning the remaining funds and avoiding reentrancy attacks